frontend/src/components/NewShoot/NewShootInfrastructureDetails.vue

```diff
@@ -190,6 +270,31 @@ const validations = {
       return this.infrastructureKind === 'vsphere'
     }),
     includesKey: includesIfAvailable('default', 'allLoadBalancerClassNames')
+  },
+  partitionID: {
+    required: requiredIf(function () {
+      return this.infrastructureKind === 'metal'
```

Comment:
We need to check if we can move provider specific code to an own component without having to duplicate things. This way we can introduce additional infra providers without the fear of breaking existing providers. WDYT?
Of course, nothing we would do within this PR.

Reply:
Yes, we already discussed this briefly. I think this would be feasible.

Reply:
👍